WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/AddCustomAmountView.swift

```diff
@@ -36,6 +36,7 @@ struct AddCustomAmountView: View {
                         }
                         .buttonStyle(PrimaryButtonStyle())
                         .disabled(viewModel.shouldDisableDoneButton)
+                        .accessibilityIdentifier("new-order-add-custom-amount-view-add-custom-amount-button")
```

Comment:
@jostnes, I'm for moving it to an enum, I'll do this shortly and will let you know 👍 

However, when it comes to naming, the proposed name will create some ambiguity, because [there's another](https://github.com/woocommerce/woocommerce-ios/blob/4c24a288e4a0223e3cb2abd2d8a89cae4d2cbdc8/WooCommerce/UITestsFoundation/Screens/Orders/UnifiedOrderScreen.swift#L29) `Add custom amount` button on the "New Order" screen which is actually used to show the `Add custom amount` view. Since both of these "screens" are technically the same screen (I created a separate `AddCustomAmountScreen` file just for comfort, it's not a separate screen per se), and since the ID for the "other" button is `new-order-add-custom-amount-button`, it felt logical to follow the naming pattern and reflect that the button in question belongs to "New Order" screen, and its "Add Custom Amount" view.

Reply:
828d7c1fcf4630ebf98538bb0db75bb7d73a5c03 moves the accessibility identifier to an enum. Let's wait for CI...